boards/x86/ehl_crb/Kconfig.defconfig

```diff
@@ -13,6 +13,13 @@ config BUILD_OUTPUT_STRIPPED
 config MP_NUM_CPUS
 	default 2
 
+if PCIE_MSI_MULTI_VECTOR
+
+config INTEL_VTD_ICTL
+	default y
+
+endif # PCIE_MSI_MULTI_VECTOR
+
```

Comment:
Is it intentional that only certain boards default to enabling INTEL_VTD_ICTL when PCIE_MSI_MULTI_VECTOR?

If the actual desired behaviour is that for all boards, where (ACPI && X86 && 64BIT && PCIE_MSI_MULTI_VECTOR) is true, that INTEL_VTD_ICTL is enabled by default. Then it is better to have a default y in the original "declaration" of INTEL_VTD_ICTL.

Reply:
True, but for qemu_x86_64. So far the requirement was for ehl, but indeed it can be generalized.

Reply:
Ah, not for qemu_x86_64, that is a shame.

But is PCIE_MSI_MULTI_VECTOR enabled on qemu_x86_64?

Reply:
VT-D cannot be used within an emulator/virtualizer. And thus multi vector MSI would need to be disabled in qemu, we are not going to need this there at least on x86/x86_64.
You need multi-vector MSI only on highly specific cases. Nowadays, usually at least, MSI-X is used (it's more flexible, more efficient...)

Reply:
Great, then it should be safe with a default y in the original config :)